### PR TITLE
NT;RE assets take precedence over OG NT assets.

### PR DESCRIPTION
**Pull request by Agiel**
Not the prettiest solution but it seems to be working.

### CODE REVIEW

**Review by nullsystem — APPROVED** [review]
Tested on Linux and Windows, looks fine by me.

File: mp/game/neo/cfg/autoexec.cfg
Comment (Rainyan) [inline mp/game/neo/cfg/autoexec.cfg:1]: Should we also drop the `neo_config.cfg` and `neo_config_custom.cfg` files from `mp/game/neo/cfg`, as we aren't using these anymore?

**Review by Rainyan — APPROVED** [review]
LGTM

File: mp/game/neo/cfg/autoexec.cfg
Comment (Agiel) [reply mp/game/neo/cfg/autoexec.cfg:1]: You're right. I'll remove them.

**Pull request merged**